src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -650,6 +652,21 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
       {errors.iframeUrl && (
         <span className="input-error">{errors.iframeUrl.message}</span>
       )}
+      <label
+        htmlFor={"autoPlaycheckbox"}
+        className={`checkbox ${
+          !watch("autoPlay", false) && "checkbox-checked"
+        }`}
+      >
+        If you are embedding a video, do you want to disable autoplay?
+      </label>
+      <input
+        type="checkbox"
+        id={"autoPlaycheckbox"}
+        name={"autoPlay"}
+        defaultChecked={!values.autoPlay}
+        ref={register}
+      />
```

Comment:
There should be `Toggler` components in the same form which this input can copy from. Example:

```tsx
     <div className="toggle-room">
      <h4 className="italic input-header">Show Schedule</h4>
      <Toggler
        name="showSchedule"
        forwardedRef={register}
        defaultToggled={DEFAULT_SHOW_SCHEDULE}
      />
```

Reply:
fixed in 418cdcb4763615a45870bc6b1dfd26c2ec198a84